.github/workflows/checks.yml

```diff
@@ -11,59 +11,59 @@ on:
       - 'cookbook/docs/**'
 jobs:
 
-  # Lists all directories that have Dockerfile in them...
-  list:
-    runs-on: ubuntu-latest
-    name: "Create a list of packages"
-    steps:
-      - uses: actions/checkout@v2
-      - name: "Provide the list"
-        id: create-list
-        run: echo "::set-output name=packages::$(find cookbook -mindepth 1 -maxdepth 5 -type f -name Dockerfile -exec dirname '{}' \; | grep -v "cookbook/examples"  | sort | jq --raw-input . | jq --slurp . | jq -c .)"
-    outputs:
-      packages: "${{ steps.create-list.outputs.packages }}"
+  # # Lists all directories that have Dockerfile in them...
```

Comment:
Is there any specific reason why these lines are being commented?

Reply:
will clean this up in the next pr